modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectDefinitionExportImportTest.java

```diff
@@ -65,6 +80,119 @@ public void testExportImportObjectDefinition() throws Exception {
 			"test-object-definition.portuguese-locale.json",
 			"test-object-definition.site-default-locale.json",
 			"TESTOBJECTDEFINITIONPORTUGUESE", "TestObjectDefinitionPortuguese");
+
+		// Published object definition
+
+		String objectDefinitionPublishedJSONString =
+			_jsonFactory.createJSONObject(
+				_defaultObjectDefinitionJSONString
+			).put(
+				"active", true
+			).put(
+				"externalReferenceCode", "TESTOBJECTDEFINITIONPUBLISHED"
+			).put(
+				"name", "TestObjectDefinitionPublished"
+			).put(
+				"status",
+				_jsonFactory.createJSONObject(
+				).put(
+					"code", 0
+				).put(
+					"label", "approved"
+				).put(
+					"label_i18n", "Approved"
+				)
+			).toString();
+
+		testExportImportJSONString(
+			objectDefinitionPublishedJSONString,
+			objectDefinitionPublishedJSONString,
+			"TESTOBJECTDEFINITIONPUBLISHED", "TestObjectDefinitionPublished");
+
+		// Cannot modify the status of published object definition
+
+		String objectDefinitionDraftJSONString = _jsonFactory.createJSONObject(
+			objectDefinitionPublishedJSONString
+		).put(
+			"status",
+			_jsonFactory.createJSONObject(
+			).put(
+				"code", 2
+			).put(
+				"label", "draft"
+			).put(
+				"label_i18n", "Draft"
+			)
+		).toString();
+
+		Class<?> clazz = getClass();
+
+		testFailedImportJSONString(
+			objectDefinitionDraftJSONString,
+			StringUtil.read(
+				clazz.getResourceAsStream(
+					"dependencies/test-object-definition.draft.error-message." +
+						"json")),
+			"TESTOBJECTDEFINITIONPUBLISHED", "TestObjectDefinitionPublished");
+
+		// Root object definition with node
```

Comment:
I would say something similar to: 
// import bound object definitions one by one

Reply:
and we would not need the second comment: `// Bound object definition`

Reply:
But we would need to pattern the comment. For example: add the word import in the previous comments :grimacing: